docs/connect.html

```diff
@@ -74,6 +74,7 @@ <h4><a id="connect_running" href="#connect_running">Running Kafka Connect</a></h
         <li><code>config.storage.topic</code> (default <code>connect-configs</code>) - topic to use for storing connector and task configurations; note that this should be a single partition, highly replicated, compacted topic. You may need to manually create the topic to ensure the correct configuration as auto created topics may have multiple partitions or be automatically configured for deletion rather than compaction</li>
         <li><code>offset.storage.topic</code> (default <code>connect-offsets</code>) - topic to use for storing offsets; this topic should have many partitions, be replicated, and be configured for compaction</li>
         <li><code>status.storage.topic</code> (default <code>connect-status</code>) - topic to use for storing statuses; this topic can have multiple partitions, and should be replicated and configured for compaction</li>
+        <li><code>plugin.path</code> (default <code>empty</code>) - a list of paths that contain plugins (connectors, converters, transformations). For the purpose of quick starts users will have to add the path that contains the FileStreamSourceConnector and FileStreamSinkConnector packaged in <code>connect-file-"version".jar</code>, because these connectors are not included by default to the <code>CLASSPATH</code> or the <code>plugin.path</code> of the Connect worker</li>
```

Comment:
Should we add the same to the Standalone section above?

Also we should update `quickstart.html` too.

Reply:
Thanks for the comment @mimaison 
I've moved this bullet further up where we include properties for both standalone and distributed. 

Regarding the quickstart, the reason I didn't change this file is that it's not shown anymore. Which might be something to fix, but possibly not in this PR.

Reply:
Thanks @kkonstantine. 

Regarding the quickstart, it's currently not shown on the website, but this [commit](https://github.com/apache/kafka/commit/c0b2afb353c9b2bd8e4fccd40c0655670f75cedc) added it back. So once we sync the docs for 3.2, it should be back.

Reply:
Yeah, I agree that it'd be good to add this information to the Connect section of the quickstart, since that's been added back for 3.2.

Reply:
Thanks for the link @mimaison. In that case, it's worth updating the quick start here too. Let me do that